flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -225,4 +225,9 @@ function get_host_machine_address {
     fi
 }
 
+function set_config_for_kubernetes {
+    set_config_key "rest.address" "0.0.0.0"
+    set_config_key "rest.bind-address" "0.0.0.0"
```

Comment:
Don't the e2e tests also use the dev-master images? If so, then this shouldn't be necessary.

Reply:
This wasn't the behaviour I observed in the kubernetes session tests - it looked like it was creating the configmap from the value locally and not in the docker image. I think its happening here: https://github.com/apache/flink/blob/master/flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/decorators/FlinkConfMountDecorator.java#L102

I verified that the config was correctly set in the docker container, but it still said in the logs `Loading configuration property: rest.bind-address, localhost`

Reply:
ah right, wans't it overriding everything with the config in the distribution? if so, that implies that... native-kubernetes? users have to configure it explicitly?

Reply:
I believe so, yes